server/src/services/home/api.js

```diff
@@ -0,0 +1,9 @@
+const health = (req, res) => {
```

Comment:
Why you have health route for every type of router?

Reply:
To check if the service is available/reachable or not :smile: